tools/install-powershell.ps1

```diff
@@ -26,6 +26,9 @@ param(
     [Parameter()]
     [switch] $Daily,
 
+    [Parameter()]
+    [switch] $UseMSI,
```

Comment:
`UseMSI` is ignored with the `Daily` Switch, the two should be on different parameter sets

Reply:
Similar issue with `destination`, `AddToPath`, `DoNotOverwrite`, but I find this less confusing in how it is currently implemented.

Reply:
Fixed